tests/components/test_remember_the_milk.py

```diff
@@ -0,0 +1,42 @@
+"""Tests for the Remember The Milk component."""
+
+import unittest
+import logging
+import os
+import homeassistant.components.remember_the_milk as rtm
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class TestConfiguration(unittest.TestCase):
+    """Basic tests for the class RememberTheMilkConfiguration."""
+
+    @staticmethod
+    def _delete_config_file():
+        if os.path.exists(rtm.CONFIG_FILE_NAME):
```

Comment:
Please mock out all IO inside the tests so that the file never even get created. I/O in tests is very very slow.

Reply:
OK